wrappers/services.go

```diff
@@ -159,7 +209,12 @@ func StartServices(apps []*snap.AppInfo, inter interacter, tm timings.Measurer)
 			}
 
 			timings.Run(tm, "start-socket-service", fmt.Sprintf("start socket service %q", socketService), func(nested timings.Measurer) {
-				err = sysd.Start(socketService)
+				switch app.DaemonScope {
```

Comment:
we need to have the switch outside so that the timings can specify "user socket services"